packages/ringcentral-widgets-demo/dev-server/containers/App/index.js

```diff
@@ -65,13 +65,13 @@ export default function App({
                   defaultOffsetX={0}
                   defaultOffsetY={73}
                   hidden={(
-                    routerProps.location.pathname === '/calls/active' ||
-                    routerProps.location.pathname.indexOf('/conferenceCall/callsOnhold') === 0 ||
-                    routerProps.location.pathname.indexOf('/conferenceCall/dialer') === 0 ||
-                    routerProps.location.pathname === '/conferenceCall/participants'
+                    routerProps.location.pathname.indexOf('/calls/active') === 0
+                    || routerProps.location.pathname.indexOf('/conferenceCall/dialer') === 0
+                    || routerProps.location.pathname.indexOf('/conferenceCall/callsOnhold') === 0
+                    || routerProps.location.pathname.indexOf('/conferenceCall/participants') === 0
                   )}
-                  goToCallCtrl={() => {
-                    phone.routerInteraction.push('/calls/active');
+                  goToCallCtrl={(sessionId) => {
```

Comment:
give it a fall back, there is a chance that other apps don't want this kind of router